test/modules/linear_boltzmann_solvers/transport_steady/transport_3d_2_unstructured_restart.lua

```diff
@@ -0,0 +1,153 @@
+vol0 = logvol.RPPLogicalVolume.Create({infx=true, infy=true, infz=true})
+mesh.SetMaterialIDFromLogicalVolume(vol0,0)
+
+vol1 = logvol.RPPLogicalVolume.Create
+({ xmin=-0.5,xmax=0.5,ymin=-0.5,ymax=0.5, infz=true })
+mesh.SetMaterialIDFromLogicalVolume(vol1,1)
+
+--############################################### Add materials
+materials = {}
+materials[1] = mat.AddMaterial("Test Material");
+materials[2] = mat.AddMaterial("Test Material2");
+
+num_groups = 1
+mat.SetProperty(materials[1], TRANSPORT_XSECTIONS, OPENSN_XSFILE, "simple_scatter_sgt1000_sigs999.9.xs")
+mat.SetProperty(materials[2], TRANSPORT_XSECTIONS, OPENSN_XSFILE, "simple_scatter_sgt1000_sigs999.9.xs")
+
+src={}
+for g=1,num_groups do
+  src[g] = 0.5
+end
+
+mat.SetProperty(materials[1], ISOTROPIC_MG_SOURCE, FROM_ARRAY, src)
+mat.SetProperty(materials[2], ISOTROPIC_MG_SOURCE, FROM_ARRAY, src)
+
+--############################################### Setup Physics
+pquad0 = aquad.CreateProductQuadrature(GAUSS_LEGENDRE_CHEBYSHEV, 2, 2)
+
+lbs_block =
+{
+  num_groups = num_groups,
+  groupsets =
+  {
+    {
+      groups_from_to = {0, 0},
+      angular_quadrature_handle = pquad0,
+      --angle_aggregation_type = "single",
+      angle_aggregation_num_subsets = 1,
+      groupset_num_subsets = 1,
+      inner_linear_method = "gmres",
+      l_abs_tol = 1.0e-6,
+      l_max_its = 10000,
+      gmres_restart_interval = 100,
+    },
+  }
+}
+bsrc={}
+for g=1,num_groups do
+  bsrc[g] = 0.0
+end
+bsrc[1] = 1.0/4.0/math.pi;
+lbs_options =
+{
+  boundary_conditions = { { name = "zmax", type = "isotropic",
+                            group_strength=bsrc}},
+  scattering_order = 1,
+  save_angular_flux = true,
+  read_restart_data = true,
+  read_restart_directory_name = "transport_3d_2_unstructured_restart", 
+  read_restart_file_stem = "transport_3d_2_unstructured"
+}
+
+phys1 = lbs.DiscreteOrdinatesSolver.Create(lbs_block)
+lbs.SetOptions(phys1, lbs_options)
+
+--############################################### Initialize and Execute Solver
+ss_solver = lbs.SteadyStateSolver.Create({lbs_solver_handle = phys1})
+
+solver.Initialize(ss_solver)
+solver.Execute(ss_solver)
+
+--############################################### Get field functions
+fflist,count = lbs.GetScalarFieldFunctionList(phys1)
+
+--############################################### Slice plot
+--slices = {}
+--for k=1,count do
+--    slices[k] = fieldfunc.FFInterpolationCreate(SLICE)
+--    fieldfunc.SetProperty(slices[k],SLICE_POINT,{x = 0.0, y = 0.0, z = 0.8001})
+--    fieldfunc.SetProperty(slices[k],ADD_FIELDFUNCTION,fflist[k])
+--    --fieldfunc.SetProperty(slices[k],SLICE_TANGENT,{x = 0.393, y = 1.0-0.393, z = 0})
+--    --fieldfunc.SetProperty(slices[k],SLICE_NORMAL,{x = -(1.0-0.393), y = -0.393, z = 0.0})
+--    --fieldfunc.SetProperty(slices[k],SLICE_BINORM,{x = 0.0, y = 0.0, y = 1.0})
+--    fieldfunc.Initialize(slices[k])
+--    fieldfunc.Execute(slices[k])
+--    fieldfunc.ExportToPython(slices[k])
+--end
```

Comment:
Maybe remove this commented out section altogether

Reply:
Fixed.